libr/core/cmd_anal.c

```diff
@@ -5992,6 +5982,28 @@ static void cmd_anal_graph(RCore *core, const char *input) {
 			break;
 		}
 		break;
+	case 'i': // agi "import graph"
```

Comment:
put quotes around "agi", we use this to find where the commands are implemented by greeping for this expresssion which have very few false positives